modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -1729,8 +1729,10 @@ void NativeScriptLanguage::unregister_script(NativeScript *script) {
 
 			Map<String, Ref<GDNative>>::Element *G = library_gdnatives.find(script->lib_path);
 			if (G) {
-				G->get()->terminate();
-				library_gdnatives.erase(G);
+				if (G->get()->get_library()->is_reloadable()) {
```

Comment:
I guess you could group the two `if`s in one.

Reply:
No problem:)